python/lsst/ts/adamSensors/model.py

```diff
@@ -57,25 +64,25 @@ def read_voltage(self):
         volts : List of floats
             the voltages on the ADAM's input channels
         """
-        if self.simulation_mode:
-            return [0, 0, 0, 0, 0, 1.25]
 
-        else:
-            try:
-                readout = self.client.read_input_registers(0, 8, unit=1)
-                return [self.counts_to_volts(r) for r in readout.registers]
-            except AttributeError:
-                # read_input_registers() *returns* (not raises) a
-                # ModbusIOException in the event of loss of ADAM network
-                # connectivity, which causes an AttributeError when we try
-                # to access the registers field. But the whole thing is
-                # really a connectivity problem, so we re-raise it as a
-                # ConnectionException, which we know how to handle. Weird
-                # exception handling is a known issue with pymodbus so it
-                # may see a fix in a future version, which may require
-                # minor code changes on our part.
-                # https://github.com/riptideio/pymodbus/issues/298
-                raise ConnectionException
+        try:
+            readout = await self.client.read_input_registers(0, 8, unit=1)
+            return [self.counts_to_volts(r) for r in readout.registers]
+        except AttributeError:
+            # read_input_registers() *returns* (not raises) a
+            # ModbusIOException in the event of loss of ADAM network
+            # connectivity, which causes an AttributeError when we try
+            # to access the registers field. But the whole thing is
+            # really a connectivity problem, so we re-raise it as a
+            # ConnectionException, which we know how to handle. Weird
+            # exception handling is a known issue with pymodbus so it
+            # may see a fix in a future version, which may require
+            # minor code changes on our part.
+            # https://github.com/riptideio/pymodbus/issues/298
+            raise ConnectionException(
+                f"Unable to reach modbus device at {self.clientip}:\
```

Comment:
Same above...

Can you fix this line break the same way you did on the CSC?

Either put it all in one line or break the string into multiple concatenated strings

```
raise ConnectionException(
                    "Unable to connect to modbus device at "
                    f"{self.clientip}:{self.clientport}"
)
```